debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -375,7 +368,8 @@ protected Object convertJson(Column column, Field fieldDefn, Object data) {
                         r.deliver(JsonBinary.parseAsString((byte[]) data));
                     }
                     catch (IOException e) {
-                        throw new ConnectException("Failed to parse and read a JSON value on " + column + ": " + e.getMessage(), e);
+                        parsingErrorHandler.error("Failed to parse and read a JSON value on '" + column + "' value " + Arrays.toString((byte[]) data), e);
```

Comment:
We might want to use this more widely later on? But good to leave it limited to JSON for now.